spec/03-plugins/01-tcp-log/01-tcp-log_spec.lua

```diff
@@ -119,6 +192,42 @@ for _, strategy in helpers.each_strategy() do
       assert.True(is_latencies_sum_adding_up)
     end)
 
+    it("logs proper latencies (grpc)", function()
+      local tcp_thread = helpers.tcp_server(TCP_PORT) -- Starting the mock TCP server
+
+      -- Making the request
+      local ok, resp = proxy_client_grpc({
+        service = "hello.HelloService.SayHello",
+        body = {
+          greeting = "world!"
+        },
+        opts = {
+          ["-authority"] = "tcp_logging_grpc.test",
+        }
+      })
+      assert.truthy(ok)
+      assert.truthy(resp)
+
+      -- Getting back the TCP server input
+      local ok, res = tcp_thread:join()
+      assert.True(ok)
+      assert.is_string(res)
+
+      -- Making sure it's alright
+      local log_message = cjson.decode(res)
+
+      assert.True(log_message.latencies.proxy < 3000)
+
+      -- Sometimes there's a split milisecond that makes numbers not
+      -- add up by 1. Adding an artificial 1 to make the test
+      -- resilient to those.
+      local is_latencies_sum_adding_up =
+        1+log_message.latencies.request >= log_message.latencies.kong +
```

Comment:
Minor style: `1 + log_message.latencies.request` instead of `1+log_message.latencies.request`.